docs/security/authentication/azure-ad-authentication.md

```diff
@@ -209,49 +199,49 @@ If you already have Octopus user accounts and you want to enable external authen
 
 !include <admin-user>
 
-## What next? {#AzureADauthentication-Whatnext?}
+## The next steps
 
-Now you're using an external identity provider it is easy to increase your security. You could consider configuring [Multi-Factor Authentication](https://docs.microsoft.com/en-us/azure/multi-factor-authentication/multi-factor-authentication), after all Octopus Deploy has access to your production environments!
+Now you're using an external identity provider; it is easy to increase your security. You could consider configuring [Multi-Factor Authentication](https://docs.microsoft.com/en-us/azure/multi-factor-authentication/multi-factor-authentication), after all, Octopus Deploy has access to your production environments!
 
-You should also consider disabling any authentication providers you aren't using, like username and password authentication which can now be disabled since **Octopus 3.5**.
+You should also consider disabling any authentication providers you aren't using, like username and password authentication.
 
-## Troubleshooting {#AzureADauthentication-Troubleshooting}
+## Troubleshooting
 
-We do our best to log warnings to your Octopus Server log whenever possible. If you are having difficulty configuring Octopus to authenticate with Azure Active Directory, be sure to check your [server logs](/docs/support/log-files.md) for warnings.
+If you are having difficulty configuring Octopus to authenticate with Azure Active Directory, check your [server logs](/docs/support/log-files.md) for warnings.
 
-### Double and triple check your configuration {#AzureADauthentication-DoubleandTriplecheckyourconfiguration}
+### Double and triple check your configuration
 
-Unfortunately security-related configuration is sensitive to everything. Make sure:
+Unfortunately, the security-related configuration is sensitive to everything. Make sure:
 
-- you don't have any typos or copy-paste errors.
-- remember things are case-sensitive.
-- remember to remove or add slash characters as we've instructed - they matter too!
+- You don't have any typos or copy-paste errors.
+- Remember, things are case-sensitive.
+- Remember to remove or add slash characters.
 
-### Check OpenID Connect metadata is working {#AzureADauthentication-CheckOpenIDConnectmetadataisworking}
+### Check OpenID Connect metadata is working
 
-You can see the OpenID Connect metadata by going to the Issuer address in your browser adding`/.well-known/openid-configuration` to the end. In our example this would have been something like `https://login.microsoftonline.com/b91ebf6a-84be-4c6f-97f3-32a1d0a11c8a/.well-known/openid-configuration`.
+You can see the OpenID Connect metadata by going to the Issuer address in your browser, adding `/.well-known/openid-configuration` to the end. In our example, this would have been something like `https://login.microsoftonline.com/b91ebf6a-84be-4c6f-97f3-32a1d0a11c8a/.well-known/openid-configuration`.
```

Comment:
```suggestion
You can see the OpenID Connect metadata by going to the Issuer address in your browser, adding `/.well-known/openid-configuration` to the end. In our example, this would be something like `https://login.microsoftonline.com/b91ebf6a-84be-4c6f-97f3-32a1d0a11c8a/.well-known/openid-configuration`.
```